src/Model/TransportModel/tsp-mvt.f90

```diff
@@ -722,15 +701,18 @@ subroutine mvt_setup_budobj(this)
     character(len=LENMODELNAME) :: modelname1, modelname2
     character(len=LENPACKAGENAME) :: packagename1, packagename2
     character(len=LENBUDTXT) :: text
-! ------------------------------------------------------------------------------
     !
     ! -- Assign terms to set up the mover budget object
     nbudterm = this%mvrbudobj%nbudterm
     ncv = 0
-    text = '        MVT-FLOW'
     naux = 0
+    if (this%depvartype == 'CONCENTRATION') then
```

Comment:
Not a fan of this, but understand that it is a quick way to get the desired output.